uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Steps/1032.php

```diff
@@ -22,8 +22,8 @@ public function upgrade_1032() {
     ]);
     $this->up1032_disableImportMenus([
       'Import Contacts',
```

Comment:
Does this also need to be changed or is this the menu name?

Reply:
This is the menu name